src/ucp/core/ucp_request.inl

```diff
@@ -171,6 +171,16 @@
     }
 
 
+#if UCS_ENABLE_ASSERT
+#  define UCP_REQUEST_CLEAN(_req) \
+    (_req)->send.state.uct_comp.func  = \
+            (uct_completion_callback_t)ucs_empty_function_do_assert_void; \
+    (_req)->send.state.uct_comp.count = 0;
```

Comment:
maybe set also req->send.uct to ucs_empty_function_do_assert

Reply:
> maybe set also req->send.uct to ucs_empty_function_do_assert

done